src/store/initial-state.js

```diff
@@ -43,16 +47,22 @@ export const mergeLocalStorage = state => {
 };
 
 /**
- * Prepare the pipeline data part of the state. This part is separated so that it
- * can be reset without overriding user settings.
- * @param {object} props Props passed to App component
+ * Prepare the pipeline data part of the state by normalizing the raw data,
```

Comment:
In docs like this I'd consider splitting these comments over multiple more direct bullet points, rather than using full sentences in a block, to make each stand out and easier to read quickly.